.github/workflows/build-old-docs.yml

```diff
@@ -33,11 +33,18 @@ jobs:
   token-check:
     runs-on: ubuntu-latest
     steps:
-      - run: echo "Success!"
-        if: "${{ github.event.inputs.release_token }} == ${{ env.release_token }}"
+      - name: Check release token
+        id: token-check
+        run: | 
+          touch SUCCESS
+        if: inputs.release_token == env.release_token
         env:
           release_token: ${{ secrets.DOCARRAY_RELEASE_TOKEN }}
-      - id: get_versions
+      - name: Fail release token
```

Comment:
couldn't this be done in the same `step`?

Reply:
I don't think so. Because the conditional `if` will skip the step if the condition is false.
Skipping does not fail the job.

Basically we need to turn the skip into a fail. One way to go about it is to have another step that tries to read something that would be done in the skipped step and fails if the skip had occurred

Reply:
> We have to have a step (`Fail release token`) that fails if the first step (`Check release token`) wasn't ran. Otherwise it will just skip `Check release token` and still succeed.